index.js

```diff
@@ -9,10 +9,61 @@ function ask(questionText) {
 
 start();
 
+let min = 1;
+let max = 100;
+let guess;
+let secretNumber;
+
 async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
+  guess = Math.floor((max - min) / 2);
+  let answer = await ask("Is this your number? Please answer (Y)es, (H)igher, (L)ower")
+  while (answer != 'Y'){
+    if (answer === 'Y') {
+      console.log('Correct!') 
+    } else if (answer === 'H') {
+      console.log("Ok, I'll guess higher...")
+    } else if (answer === 'L') {
+      console.log("Ok, I'll guess lower...")
+    } else {
+      console.log("I don't understand that, please enter again.")
+    }
```

Comment:
you should prompt the user with another await ask outside of you else statement. Leaving it as is will cause an infinite loop, due to the local variable answer never changing throughout the loop

Reply:
Additionally, you should add this line to the end of your while loop to avoid an infinite loop error:
```js
answer = await ask("Is this your number? Please answer (Y)es, (H)igher, (L)ower")
```

that way, the user will always be prompted with a question.